controllers/csiaddons/csiaddonsnode_controller.go

```diff
@@ -51,6 +51,7 @@ type CSIAddonsNodeReconciler struct {
 	ConnPool *connection.ConnectionPool
 }
 
+//+kubebuilder:rbac:groups=coordination.k8s.io,resources=leases,verbs=get
```

Comment:
```suggestion
```
We don't need to find leader in csi-addons-node controller right ?

Reply:
maybe not